Tasks5/Postfix_calculator/Postfix_calculator.c

```diff
@@ -0,0 +1,16 @@
+﻿#define _CRT_SECURE_NO_WARNINGS
+
+#include <stdio.h>
+#include <stdlib.h>
+#include "stack.h"
+
+int main(void)
+{
```

Comment:
Скобки у нас по стайлгайду в стиле BSD, так:
```suggestion
int main(void) {
```